tutorial/chapter1.md

```diff
@@ -324,7 +341,7 @@ Along with the features already described in the [kpsr-tutorial - chapter 1](htt
     - `kpsr::vision_ocv::FileImageStreamingService`: reads images from a folder and publishes them.
     - [`kpsr_tutorial::ImageProcessingService`](../modules/image_processing/include/image_processing/image_processor_service.h): Simulates an image processing algorithm which calculates the pixel mean of the image. After that, each service will publish a data which contains this mean.
 
-- Note that there are five `kpsr_tutorial::ImageProcessingService`s to show the differences in the behaviour of threads and Data Multiplexer features. So:
+- Note that there are five `kpsr_tutorial::ImageProcessingServices` to show the differences in the behaviour of threads and Data Multiplexer features. So:
     - Each subscriber of these services (Data Multiplexer Consumers) is handled by one thread. Therefore, there are five threads associated to these consumers.
     - The second argument represents the complexity of an image processing algorithm as a delay in milliseconds added to the task executed by these services. A more complex algorithm is simulated by a longer running task. Due to the features of the Data Multiplexer, the service with higher delay will process fewer images (because it will spend more time processing and discard the images it can't receive). In this example the `imageProcessingService_1` will receive more images than any other service (it has the lowest delay: 50 [ms]), whereas the `imageProcessingService_2` will receive few images than any other service (it has the highest delay: 10000 [ms]).
```

Comment:
Maybe blending this explanation (from 344 to 346 included) with the explanation of the results would flow better.